global.css

```diff
@@ -0,0 +1,76 @@
+body {
+    font-family: "Fellix", sans-serif;
```

Comment:
indentations -- please fix inconsistent indents, we use only tabs

Reply:
I have fixed this... but as regards the font family, I couldn't find that on Google fonts.

Reply:
okay.

let's remove the "Fellix" font from this branch.

1. we'll make namaka, as a library, free of any fonts.  
    any application consuming namaka will be able to specify any font they like, with  
    `html { font-family: "Fellix", sans-serif; }`
3. i'm also unsure of fellix's licensing